packages/react/src/components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -43,35 +43,42 @@ export const ProgressStep = ({ ...props }) => {
     }
   };
 
-  const SVGIcon = ({ complete, current, description, invalid, prefix }) => {
+  const SVGIcon = ({
+    complete,
+    current,
+    description,
+    invalid,
+    prefix,
+    ...rest
+  }) => {
     if (invalid) {
-      return <Warning16 className={`${prefix}--progress__warning`} />;
+      return <Warning16 className={`${prefix}--progress__warning`} {...rest} />;
     }
     if (current) {
       return (
-        <svg>
+        <svg {...rest}>
           <path d="M 7, 7 m -7, 0 a 7,7 0 1,0 14,0 a 7,7 0 1,0 -14,0" />
           <title>{description}</title>
         </svg>
       );
     }
     if (complete) {
       return (
-        <CheckmarkOutline16 role="img">
+        <CheckmarkOutline16 role="img" {...rest}>
```

Comment:
Since the icons are aria-hidden by default, i believe, instead of adding `{...rest}` it might be better to remove `role="img"` and the `aria-label` added to SVGIcon. Since `label` is used twice, VO reads it twice and no longer reads the description. This change also removes the DAP violation.

Reply:
removed the `role` attribute and reverted aria-label changes